.azure-pipelines/scripts/sap_hana/linux/50_increase_docker_memory.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -ex
+
+sudo sysctl -w vm.max_map_count=$(expr 6 \* 65536)
```

Comment:
Could you add a comment about why we are using `6 * 65536` ?

Reply:
Good call - made me realise that `6 * 65536` actually corresponds to 3GB, not 6GB.

I changed the computation to make it correspond to 6GB, knowing that HANA may not end up using all that RAM (since the previous builds here actually seemed to run fine with 3GB).

Reply:
👍 thx